src/app/app.component.ts

```diff
@@ -1,8 +1,26 @@
 import { Component } from "@angular/core";
+import { DialogType } from "projects/core/src/lib/models/dialogType";
+import { DialogService } from "projects/core/src/lib/services/dialog.service";
 
 @Component({
 	selector: "jsmu-root",
 	templateUrl: "./app.component.html",
 	styleUrls: ["./app.component.scss"]
 })
-export class AppComponent {}
+export class AppComponent {
+	public dialogType!: DialogType;
+	public isOpen!: boolean;
+
+	constructor(private dialogService: DialogService) {
+		dialogService.$dialogType.subscribe((dialogType: DialogType) => {
+			this.dialogType = dialogType;
+		});
+		dialogService.$isOpen.subscribe((isOpen: boolean) => {
+			this.isOpen = isOpen;
+		});
+	}
```

Comment:
constructor in Angular use as the di container, for this logic more suitable ngOnInit

Reply:
done